src/oop/oop2.py

```diff
@@ -3,11 +3,14 @@
 # Also change it so the num_wheels defaults to 4 if not specified when the
 # object is constructed.
 
+
 class GroundVehicle():
-    def __init__(self, num_wheels):
-        self.num_wheels = num_wheels
+    def __init__(self):
+        self.num_wheels = 4
```

Comment:
this will set all ground vehicles to 4 wheels no matter what. How would you accommodate a 6-wheeled ground vehicle?